src/main/proto/pb.proto

```diff
@@ -1009,16 +1009,24 @@ message PersistableEnvelope {
         UserPayload user_payload = 10;
         PaymentAccountList payment_account_list = 11;
 
-        CompensationRequestPayload compensation_request_payload = 12; // deprecated. Not used anymore.
```

Comment:
Doesn't this break backwards compatibility?

Reply:
It is not used by Bisq users, so should have no effect. At least with my tests between old and new version I did not see any issue.

Reply:
Ah, that's true, this is not out yet and more reorg here is to be expected.